jekyll/_cci2/server-3-operator-proxy.adoc

```diff
@@ -0,0 +1,59 @@
+---
+version:
+- Server v3.x
+- Server Admin
+---
+= CircleCI Server v3.x Configuring a Proxy
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+Depending on your security requirements, you might want to install CircleCI server behind a proxy. Installing behind a proxy gives you the power to monitor and control access between your installation and the broader internet.
+
+toc::[]
+
+== Installation and configuration
+There are two stages to installing CircleCI server behind a proxy. First at the point of installation the proxy addresses need to be specified, along with any addresses that should not be behind the proxy.
+
+=== Installing behind a proxy
+The installation process is described in detail in the https://circleci.com/docs/2.0/server-3-install/[CircleCI Server v3.x Installation guide]. Both proxy and non-proxy addresses should be supplied using the arguments described https://kots.io/kotsadm/installing/online-install/#proxies[here]. The installation command should be in the form:
+
+[source,bash]
+----
+kubectl kots install circleci-server --http-proxy <my-http-proxy-uri> --https-proxy <my-https-proxy> --no-proxy <my-no-proxy-list>
```

Comment:
Can we take out the HTTPS argument here due to the known issue? @nsmith5

Reply:
Nope. See https://circleci.atlassian.net/browse/SERVER-1400?focusedCommentId=158461

Reply:
thank you!